src/features/messages/queries/useMessageQuery.ts

```diff
@@ -21,6 +22,11 @@ export function isValidSearchQuery(input: string) {
   return !!searchValueToPostgresBytea(input);
 }
 
+export function isValidDomainId(domainId: string | null, multiProvider: MultiProvider) {
+  if (!domainId) return false;
+  return !!multiProvider.tryGetDomainId(domainId);
```

Comment:
```suggestion
  return multiProvider.hasChain(domainId);
```

Reply:
Didn't know there was a function for this, thanks!